sensing_node:
  ros__parameters:
    use_sim_time: true
    aruco_ids: [201, 202, 203, 204, 205, 206, 207, 208, 209, 210, 211, 212, 213, 214, 215, 216, 301, 302, 303, 304, 305, 306, 307, 308, 309, 310, 311, 312, 313, 314, 315, 316]

    yaml_output_path: "/tmp/chess_configuration_final.yaml"